migrations/20230725162810-add_title_to_stories_table.sql

```diff
@@ -0,0 +1,9 @@
+-- +migrate Up
+
+ALTER TABLE stories
+ADD COLUMN title VARCHAR(235);
```

Comment:
Why do we want to limit the title length at the DB level? If it is about the rendering as discussed offline, then this logic should just be in the frontend, not the DB.